packages/mixins/apollo-element-mixin.d.ts

```diff
@@ -1,9 +1,10 @@
 import ApolloClient, { ApolloError } from 'apollo-client';
 import { DocumentNode } from 'graphql';
+import { LitElement } from 'lit-element';
```

Comment:
The mixin packages shuold not reference lit-element, as they are used for a variety of packages. we'll need to find another solution.